src/main/java/com/amazon/opendistroforelasticsearch/jdbc/protocol/JdbcQueryRequest.java

```diff
@@ -22,13 +22,19 @@
 public class JdbcQueryRequest implements QueryRequest {
 
     String statement;
-
+    int fetchSize;
```

Comment:
Make it `private` and `final` if possible?

Reply:
Made it private, it is effectively final.